libraries/AP_AdvancedFailsafe/AP_AdvancedFailsafe.cpp

```diff
@@ -158,6 +158,27 @@ const AP_Param::GroupInfo AP_AdvancedFailsafe::var_info[] = {
     // @Units: km
     AP_GROUPINFO("MAX_RANGE",   20, AP_AdvancedFailsafe, _max_range_km,    0),
     
+    // @Param: TERM_POS_AIL
+    // @DisplayName: Aileron Aeroterminate Position
+    // @Description: This is the position the aileron will go to during an aerotermination
+    // @User: Advanced
+    // @Units: Percentage (-100 to 100, default 100)
+    AP_GROUPINFO("TERM_POS_AIL",   23, AP_AdvancedFailsafe, _term_pos_ail,    100),
```

Comment:
index should be 21.
Units should be ```// @Units: %```
Add ```// @Range: -100 100```
Description should mention that 0 means neutral deflection and could also mention the default
Display name would be better to say ``Aeroterminate Position for Aileron'' so Aileron is at the end so the param names and descriptions only have that one attribute different on the end.

Reply:
Also, the word "Aeroterminate" is new. Let's keep the vocabulary consistent and name it ```Terminate Aileron Position```